server/src/core/error.ts

```diff
@@ -18,7 +18,8 @@ export type ErrorCode =
   | 'invalidToken'
   | 'resourceNotFound'
   | 'uniquenessFailed'
-  | 'serverError';
+  | 'serverError'
+  | 'nominateFailed';
```

Comment:
I wonder whether t makes sense to have a specific error code for this. Meanwhile, rename it to `nomationFailed`.